### PR TITLE
[EDU-4498] feat: add bot management architecture

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

- Added diagram image
- Added architecture text

### Additional links

Compare with original version: https://docs.google.com/document/d/1tNfvf2wpQj_F39jjYi7FWTX4qSovIwB0LaZVxPQEJIU/edit

[EDU-4498]: https://aziontech.atlassian.net/browse/EDU-4498?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/architectures/bot-management-architecture.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: Protect Your Edge Applications with a Bot Management Solution
+description: Use Azion Bot Management to protect edge applications from bots, preventing malicious attacks.
+meta_tags: architecture, secure, bot, bot management, security, firewall
+namespace: docs_arch_bot_management
+permalink: /documentation/architecture/bot-management/
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/architectures/bot-management-architecture.mdx:6]:
```suggestion
permalink: /documentation/architecture/bot-management/protect-your-applications-with-bot-management/
```

Comment (gabriel-azion) [reply src/content/docs/en/pages/architectures/bot-management-architecture.mdx:6]: sei que fica um pouco grande, mais para caso termos mais archs de bot. No caso do jamstack por exemplo, teremos mais que uma, ai ficará algo como `/documentation/architecture/jamstack/deploy-ssg-jamstack-applications/`

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- Usando [Azion Templates](/pt-br/documentacao/produtos/marketplace/templates/), projetos pré-construídos e pré-configurados projetados para acelerar a implementação do gerenciamento de bots:
+    - [Bot Manager Starter Kit](/pt-br/documentacao/produtos/guias/bot-manager-starter-kit/): com este template, você pode implantar um novo stack de edge e ver o Bot Manager em ação.
+    - [Bot Manager Integration Kit](/pt-br/documentacao/produtos/guias/bot-manager-integration-kit/): uma maneira fácil de integrar a função do Bot Manager em uma instância existente do Edge Firewall da Azion.
+    - [Bot Manager and TOR Block Starter Kit](/pt-br/documentacao/produtos/guias/bot-manager-and-tor-block-starter-kit/): implante um stack básico de segurança para proteger seus domínios de bots maliciosos e requisições de TOR Exit Nodes.
+
+- Do zero: crie um stack de edge inteiro do zero, incluindo a integração do Bot Manager. Para fazer isso:
+    1. [Crie uma edge application](/pt-br/documentacao/produtos/guias/build/criar-uma-aplicacao/).
+    2. [Adicione um domínio personalizado a ela](/pt-br/documentacao/produtos/guias/configurar-dominio/).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx:50]:
```suggestion
    2. [Adicione um domínio personalizado à edge application](/pt-br/documentacao/produtos/guias/configurar-dominio/).
```

**Review by GabrielAzion — APPROVED** [review]

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/architectures/bot-management-architecture.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: Protect Your Edge Applications with a Bot Management Solution
+description: Use Azion Bot Manager to protect edge applications from bots, preventing malicious attacks.
+meta_tags: architecture, secure, bot, bot management, security, firewall
+namespace: docs_arch_bot_management
+permalink: /documentation/architecture/bot-management/protect-your-applications-with-bot-management/
+---
+
+Azion [bots management solution]() utilizes machine learning to analyze request behaviors and intentions, distinguishing between humans and various bot categories. Upon analysis, each request is scored and classified, allowing for the execution of predefined preventive actions based on customer-set thresholds. This dynamic approach is enhanced by continuous data analysis, enabling the refinement of detection rules for improved efficiency and security. This solution ensures seamless user experiences while effectively countering malicious bot activity.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/bot-management-architecture.mdx:9]:
```suggestion
Azion [bots management solution](https://www.azion.com/en/solutions/bot-management/) utilizes machine learning to analyze request behaviors and intentions, distinguishing between humans and various bot categories. Upon analysis, each request is scored and classified, allowing for the execution of predefined preventive actions based on customer-set thresholds. This dynamic approach is enhanced by continuous data analysis, enabling the refinement of detection rules for improved efficiency and security. This solution ensures seamless user experiences while effectively countering malicious bot activity.
```

File: src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+---
+title: Proteja suas edge applications com uma solução de gerenciamento de bots
+description: Utilize o Azion Bot Manager para proteger edge applications de bots, prevenindo ataques maliciosos.
+meta_tags: arquitetura, secure, bot, bot management, security, firewall
+namespace: docs_arch_bot_management
+permalink: /documentacao/arquitetura/gerenciamento-de-bots/proteja-edge-applications-com-bot-management/
+---
+
+A solução de [gerenciamento de bots da Azion]() utiliza aprendizado de máquina para analisar comportamentos e intenções das requisições, distinguindo entre humanos e várias categorias de bots. Após a análise, cada requisição é pontuada e classificada, permitindo a execução de ações preventivas predefinidas com base em limites definidos pelo cliente. Essa abordagem dinâmica é aprimorada por análises contínuas de dados, possibilitando o aperfeiçoamento das regras de detecção para maior eficiência e segurança. Essa solução garante experiências de usuário perfeitas ao combater efetivamente atividades maliciosas de bots.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx:9]:
```suggestion
A solução de [gerenciamento de bots da Azion](https://www.azion.com/pt-br/solucoes/gerenciamento-de-bots/) utiliza aprendizado de máquina para analisar comportamentos e intenções das requisições, distinguindo entre humanos e várias categorias de bots. Após a análise, cada requisição é pontuada e classificada, permitindo a execução de ações preventivas predefinidas com base em limites definidos pelo cliente. Essa abordagem dinâmica é aprimorada por análises contínuas de dados, possibilitando o aperfeiçoamento das regras de detecção para maior eficiência e segurança. Essa solução garante experiências de usuário perfeitas ao combater efetivamente atividades maliciosas de bots.
```

File: src/content/docs/en/pages/architectures/bot-management-architecture.mdx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- [Edge Application](/en/documentation/products/build/edge-application/) is your application running on top of the Azion Edge Platform.
+- [Data Stream](/en/documentation/products/observe/data-stream/) and [Real-Time Events](/en/documentation/products/observe/real-time-metrics/) enable you to monitor all Bot Manager activity.
+
+---
+
+## Implementation
+
+You can test and implement Azion Bot Manager in two ways:
+
+- Using [Azion Templates](/en/documentation/products/marketplace/templates/), pre-built and pre-configured projects designed to accelerate the implementation of bot management:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/bot-management-architecture.mdx:43]:
```suggestion
- Using [Azion Templates](/en/documentation/products/marketplace/templates/): pre-built and pre-configured projects designed to accelerate the implementation of bot management:
```

File: src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- [Edge Application](/pt-br/documentacao/produtos/build/edge-application/) é sua aplicação em execução na Plataforma de Edge da Azion.
+- [Data Stream](/pt-br/documentacao/produtos/observe/data-stream/) e o [Real-Time Events](/pt-br/documentacao/produtos/observe/real-time-metrics/) permitem que você monitore toda a atividade do Bot Manager.
+
+---
+
+## Implementação
+
+Você pode testar e implementar o Azion Bot Manager de duas maneiras:
+
+- Usando [Azion Templates](/pt-br/documentacao/produtos/marketplace/templates/), projetos pré-construídos e pré-configurados projetados para acelerar a implementação do gerenciamento de bots:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx:43]:
```suggestion
- Usando [Azion Templates](/pt-br/documentacao/produtos/marketplace/templates/): projetos pré-construídos e pré-configurados projetados para acelerar a implementação do gerenciamento de bots:
```

File: src/content/docs/en/pages/architectures/bot-management-architecture.mdx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    - [Bot Manager Starter Kit](/en/documentation/products/guides/bot-manager-starter-kit/): with this template, you're able to deploy a new edge stack and see the Bot Manager in action.
+    - [Bot Manager Integration Kit](/en/documentation/products/guides/bot-manager-integration-kit/): an easy way to integrate the Bot Manager function in an existent Edge Firewall instance on Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/bot-management-architecture.mdx:45]:
```suggestion
    - [Bot Manager Integration Kit](/en/documentation/products/guides/bot-manager-integration-kit/): an easy way to integrate the Bot Manager function in an existent edge firewall on Azion.
```

File: src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+    - [Bot Manager Starter Kit](/pt-br/documentacao/produtos/guias/bot-manager-starter-kit/): com este template, você pode implantar um novo stack de edge e ver o Bot Manager em ação.
+    - [Bot Manager Integration Kit](/pt-br/documentacao/produtos/guias/bot-manager-integration-kit/): uma maneira fácil de integrar a função do Bot Manager em uma instância existente do Edge Firewall da Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx:45]:
```suggestion
    - [Bot Manager Integration Kit](/pt-br/documentacao/produtos/guias/bot-manager-integration-kit/): uma maneira fácil de integrar a função do Bot Manager em um edge firewall existente na Azion.
```

File: src/content/docs/en/pages/architectures/bot-management-architecture.mdx [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    - [Bot Manager and TOR Block Starter Kit](/en/documentation/products/guides/bot-manager-and-tor-block-starter-kit/): deploy a basic security suite to protect your domains from bad bots and TOR Exit Nodes requests.
+
+- From scratch: create an entire edge stack from zero, including Bot Manager integration. To do so: 
+    1. [Create an edge application](/en/documentation/products/guides/build/build-an-application/).
+    2. [Add a custom domain to it](/en/documentation/products/guides/configure-a-domain/).
+    3. [Install Bot Manager integration](/en/documentation/products/guides/bot-manager/).
+    4. [Configure Bot Manager integration](/en/documentation/products/guides/bot-manager/#configuring-the-integration).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/bot-management-architecture.mdx:52]:
```suggestion
    3. [Install Bot Manager](/en/documentation/products/guides/bot-manager/).
    4. [Configure Bot Manager](/en/documentation/products/guides/bot-manager/#configuring-the-integration).
```

File: src/content/docs/en/pages/architectures/bot-management-architecture.mdx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+        - While setting up the function instance, you'll define the parameters for it.
+    5. [Monitor bot activity](/en/documentation/products/observe/data-stream/first-steps/) and calibrate rules.
+
+---
+
+## Related docs
+
+- [Understand Azion Integrations](/en/documentation/products/marketplace/integrations/)
+- [How to manage bots](/en/documentation/products/guides/secure/manage-bots/)
+- [Bot management use case]()
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/bot-management-architecture.mdx:62]:
```suggestion
```

File: src/content/docs/en/pages/architectures/bot-management-architecture.mdx [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+- [Web Application and API Security (WAAP) use case]()
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/bot-management-architecture.mdx:63]:
```suggestion
- [Web Application and API Security (WAAP)](/en/documentation/architecture/edge-firewall/web-application-and-api-protection-waap/)
```

File: src/content/docs/en/pages/architectures/bot-management-architecture.mdx [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+- [Security modernization use case]()
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/bot-management-architecture.mdx:64]:
```suggestion
```

File: src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+        - Ao configurar a instância da função, você definirá os parâmetros para ela.
+    5. [Monitore a atividade de bots](/pt-br/documentacao/produtos/observe/data-stream/primeiros-passos/) e calibre as regras.
+
+---
+
+## Documentação relacionada
+
+- [Conheça as Azion Integrations](/pt-br/documentacao/produtos/marketplace/integracoes/)
+- [Como gerenciar bots](/pt-br/documentacao/produtos/guias/secure/gerenciar-bots/)
+- [Bot management solution]()
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx:62]:
```suggestion
```

File: src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+        - Ao configurar a instância da função, você definirá os parâmetros para ela.
+    5. [Monitore a atividade de bots](/pt-br/documentacao/produtos/observe/data-stream/primeiros-passos/) e calibre as regras.
+
+---
+
+## Documentação relacionada
+
+- [Conheça as Azion Integrations](/pt-br/documentacao/produtos/marketplace/integracoes/)
+- [Como gerenciar bots](/pt-br/documentacao/produtos/guias/secure/gerenciar-bots/)
+- [Web Application and API Security (WAAP) use case](/pt-br/documentacao/arquitetura/edge-firewall/web-application-and-api-protection-waap/)
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx:62]:
```suggestion
- [Web Application and API Security (WAAP) ](/pt-br/documentacao/arquitetura/edge-firewall/web-application-and-api-protection-waap/)
```

File: src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+        - Ao configurar a instância da função, você definirá os parâmetros para ela.
+    5. [Monitore a atividade de bots](/pt-br/documentacao/produtos/observe/data-stream/primeiros-passos/) e calibre as regras.
+
+---
+
+## Documentação relacionada
+
+- [Conheça as Azion Integrations](/pt-br/documentacao/produtos/marketplace/integracoes/)
+- [Como gerenciar bots](/pt-br/documentacao/produtos/guias/secure/gerenciar-bots/)
+- [Web Application and API Security (WAAP) ](/pt-br/documentacao/arquitetura/edge-firewall/web-application-and-api-protection-waap/)
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/bot-management-arquitetura.mdx:62]:
```suggestion
- [Arquitetura de Web Application and API Security (WAAP) ](/pt-br/documentacao/arquitetura/edge-firewall/web-application-and-api-protection-waap/)
```

**Pull request merged**